adoc/MAIN.release-notes.adoc

```diff
@@ -21,20 +21,34 @@ To make such entries easier to identify, they contain a note to that effect.
 
 == Changes in 4.2.0
 
+=== Deprecations in 4.2.0
```

Comment:
@jordimassaguerpla Do all of these really apply to 4.2.0? The section heading was previously 4.2.x but that doesn't make sense in this document.

Reply:
yes, they apply to 4.2.0